imagersite/imager_profile/templates/profile.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% block content %}
+
+<body>
+<h1><big>Hello, {{ user.username }}. You joined us on {{ user.date_joined|date:"l, F dS, Y." }}</big></h1>
+<h3>Looks like you need to upload more photos!</h3>
+<a href="{% url 'library' %}"><h1 class="col-md-6"><strong style="font-size:200px;">{{ user.albums.all|length }}</strong><br>Albums</h1></a>
```

Comment:
There's a `count` api on Django querysets that is more efficient than grabbing all items and then counting the list:

```jinja2
{{ user.albums.count() }}
```